docs/developers/talawa-api/constraints/superadmin.md

```diff
@@ -0,0 +1,19 @@
+---
+id: SuperAdmin
+title: SuperAdmin
+---
+
+## Who Is An Super Administrator (`SuperAdmin`)?
+
+Super Administrators are users who have the ability to manage and moderate administrators and organizations.
```

Comment:
Let's change this to, "In addition to having all Admin permissions, Super Administrators are users who have the ability to manage and moderate administrators and organizations."   Then delete the sentence below regarding admin permissions.

Reply:
done.